examples/example_user_input_material.py

```diff
@@ -56,8 +56,10 @@
     or 'bm2' (birch-murnaghan 2nd order, if you choose to ignore temperature
        (your choice in geotherm will not matter in this case))
     or 'bm3' (birch-murnaghan 3rd order, if you choose to ignore temperature
-        (your choice in geotherm will not matter in this case))"""
-    method = 'slb3'
+        (your choice in geotherm will not matter in this case)
+    or 'vinet' (vinet equation of state, if you choose to ignore temperature
+        (your choice in geotherm will not matter in this case)))"""
+    method = 'vinet'
```

Comment:
I agree with Timo. slb3 should be the default - but of course vinet should be added to the list. 
vinet should also be added to example_compare_all_methods.py. 
I don't think there's any need to add another example... this is fine for me.